platform/admin/ui/src/main/webapp/templates/installer/finish.handlebars

```diff
@@ -26,14 +26,12 @@
             <br/>
             <br/>
             A system restart is required each time these files are modified in order for the changes to take effect.
-            {{#if modified}}
                 <br/>
                 <br/>
```

Comment:
❓ I wonder if we force the restart "now" since solr is in a corrupt state until the system is restarted.

Reply:
Good point. We may just want to keep a single `Restart` button.